### PR TITLE
Web sdk

### PR DESCRIPTION
**Pull request by albho**

### CODE REVIEW

File: binding/web/README.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+## Falcon Speaker Diarization Engine
+
+Made in Vancouver, Canada by [Picovoice](https://picovoice.ai)
+
+Falcon is an on-device speaker diarization engine. Falcon is:
+
+- Private; All voice processing runs locally.
+- Cross-Platform:
+  - Linux (x86_64), macOS (x86_64, arm64), Windows (x86_64)
+  - Raspberry Pi (4, 3) and NVIDIA Jetson Nano
```
Comment (mrrostam) [inline binding/web/README.md:12]:
```suggestion
  - Raspberry Pi (4, 3) and NVIDIA Jetson Nano
  - Chrome, Safari, Firefox, and Edge
```

Comment (mrrostam) [reply binding/web/README.md:12]: You should also update the main readme

File: binding/web/test/falcon_perf.test.ts [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
 
-const ACCESS_KEY = Cypress.env('ACCESS_KEY');
-const NUM_TEST_ITERATIONS = Number(Cypress.env('NUM_TEST_ITERATIONS'));
-const INIT_PERFORMANCE_THRESHOLD_SEC = Number(
-  Cypress.env('INIT_PERFORMANCE_THRESHOLD_SEC')
-);
-const PROC_PERFORMANCE_THRESHOLD_SEC = Number(
-  Cypress.env('PROC_PERFORMANCE_THRESHOLD_SEC')
-);
+// const ACCESS_KEY = Cypress.env('ACCESS_KEY');
```
Comment (mrrostam) [inline binding/web/test/falcon_perf.test.ts:3]: left over from debug?

Comment (albho) [reply binding/web/test/falcon_perf.test.ts:3]: yes, oops

**Review by ksyeo1010 — APPROVED** [review]

File: binding/web/src/types.ts [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+ * FalconModel types
+ */
+export type FalconModel = PvModel;
+
+export type FalconSegment = {
+  /** Start of segment in seconds. */
+  startSec: number;
+  /** End of segment in seconds. */
+  endSec: number;
+  /** The speaker tag is a non-negative integer identifying unique speakers, with `0` reserved for
```
Comment (mrrostam) [inline binding/web/src/types.ts:39]: this is not true for falcon, look at the python binding for definitions

File: demo/web/index.html [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+
+      function writeMessage(message) {
+        console.log(message);
+        document.getElementById("status").innerHTML = message;
+      }
+
+      function setSegmentsTable(segments) {
+        let html = `
+          <tr>
+            <th>startSec</th>
```
Comment (mrrostam) [inline demo/web/index.html:123]:
```suggestion
            <th>Start time (s)</th>
```
For showing the output, can we have a table similar to octopus web demo: https://github.com/Picovoice/octopus/blob/main/demo/web/index.html

**Pull request merged**